### PR TITLE
app-misc/goobook: Bump to 3.3

### PR DESCRIPTION
**Pull request by thinrope**
* support for python-3.{6,7}
* removed indirect dependencies
* EAPI=7
* fixed syntax per PR conversation
* added /self as proxy maintainer

Bug: https://bugs.gentoo.org/686384
Reported-by: Kalin KOZHUHAROV <kalin@thinrope.net>
Signed-off-by: Kalin KOZHUHAROV <kalin@thinrope.net>
Related: https://bugs.gentoo.org/678398

### CODE REVIEW

**Comment by thinrope** [comment]
When this lands properly, we can remove the older version and close its QA bug https://bugs.gentoo.org/670536

File: app-misc/goobook/goobook-3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
```
Comment (titanofold) [inline app-misc/goobook/goobook-3.3.ebuild]: 3.7 is also supported.

File: app-misc/goobook/goobook-3.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+inherit distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="Access your Google contacts from the command line"
+HOMEPAGE="https://gitlab.com/goobook/goobook"
+SRC_URI="mirror://pypi/g/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
```
Comment (titanofold) [inline app-misc/goobook/goobook-3.3.ebuild]: This SLOT is redundant and can be removed. (`SLOT=0` is assumed.)

File: app-misc/goobook/goobook-3.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (titanofold) [inline app-misc/goobook/goobook-3.3.ebuild]: Empty variables don't need to be included.

File: app-misc/goobook/goobook-3.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	>=dev-python/google-api-python-client-1.6.4[${PYTHON_USEDEP}]"
+# dev-python/{simplejson,oauth2client} are deps for the above
+
+DEPEND="${PYTHON_DEPS}
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_install() {
+	distutils-r1_src_install
+	readme.gentoo_create_doc
+}
```
Comment (titanofold) [inline app-misc/goobook/goobook-3.3.ebuild]: Move `src_install()` to be in between `DOC_CONTENTS="..."` and `pkg_postinst()`.

File: app-misc/goobook/goobook-3.3.ebuild (hunk 2 above)
Comment (thinrope) [reply app-misc/goobook/goobook-3.3.ebuild]: repoman complains:
```
  SLOT.invalid [fatal]          1
   app-misc/goobook/goobook-3.3.ebuild: SLOT: invalid value: ''
```
Also [devmanual](https://devmanual.gentoo.org/general-concepts/slotting/index.html) is explicit:
`Most packages have no need for slotting. These packages specify SLOT="0" in the ebuilds. This is not the same as specifying an empty slot (SLOT="") — an empty slot means "disable slotting entirely", and should not be used. `

Comment (titanofold) [reply app-misc/goobook/goobook-3.3.ebuild]: Huh, I wonder when that changed. I'm getting old and that wasn't a thing when I first started, so never mind.

**Comment by titanofold** [comment]
The first line of the commit message should be written: "app-misc/goobook: Bump to 3.3".

**Comment by titanofold** [comment]
Also, if you would like to assume maintainership, add yourself and Proxy to the metadata.xml.

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

**Pull request closed without merging**